sdk/search/Azure.ResourceManager.Search/api/Azure.ResourceManager.Search.netstandard2.0.cs

```diff
@@ -65,6 +65,9 @@ public partial class SearchServiceCollection : Azure.ResourceManager.ArmCollecti
     public partial class SearchServiceData : Azure.ResourceManager.Models.TrackedResourceData
     {
         public SearchServiceData(Azure.Core.AzureLocation location) : base (default(Azure.Core.AzureLocation)) { }
+        public Azure.ResourceManager.Search.Models.DataPlaneAuthOptions AuthOptions { get { throw null; } set { } }
+        public bool? DisableLocalAuth { get { throw null; } set { } }
```

Comment:
Could we rename the property to `IsLocalAuthDisabled`?

Reply:
Did you do this rename? If you did, please generate the API file again to make sure the api file synced with the source code.

Reply:
had a formatting error in the rename rule